jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt

```diff
@@ -108,3 +115,21 @@ class ContainerLogsAction(
         return true
     }
 }
+
+class ExecuteCommandAction(
+    private val project: Project,
+    private val container: ContainerDetails
+) : AnAction(message("ecs.execute_command_run"), null, null) {
+    override fun actionPerformed(e: AnActionEvent) {
+        var sessionManagerInstalled: Int = 1
+        runBlocking(Dispatchers.IO) {
+            val process = CapturingProcessHandler(GeneralCommandLine("session-manager-plugin")).runProcess()
+            sessionManagerInstalled = process.exitCode
+        }
```

Comment:
in general, avoid vars when you can cleanly use a val instead
```suggestion
        val sessionManagerExitCode = runBlocking(Dispatchers.IO) {
            val process = CapturingProcessHandler(GeneralCommandLine("session-manager-plugin")).runProcess()

            process.exitCode
        }
```